src/goto-programs/remove_vector.cpp

```diff
@@ -82,13 +82,13 @@ static void remove_vector(exprt &expr)
        expr.id()==ID_mod  || expr.id()==ID_bitxor ||
        expr.id()==ID_bitand || expr.id()==ID_bitor)
     {
+      auto const &binary_expr = to_binary_expr(expr);
```

Comment:
bitand is not binary:

https://github.com/diffblue/cbmc/blob/develop/src/util/std_expr.h#L2716